app/services/invite_schools.rb

```diff
@@ -52,9 +59,10 @@ def reference
   end
 
   def nomination_url(token)
-    Rails.application.routes.url_helpers.nominations_url(
-      token: token,
-      host: Rails.application.config.domain,
-    )
+    token
+    # Rails.application.routes.url_helpers.nominations_url( # TODO
+    #   token: token,
+    #   host: Rails.application.config.domain,
+    # )
```

Comment:
I'll come back to this in "Seb's PR" (nominate an induction tutor) and I actually have somewhere for this to go